vendor/bat-native-ads/src/bat/ads/internal/catalog/catalog.cc

```diff
@@ -6,8 +6,8 @@
 #include "bat/ads/internal/catalog/catalog.h"
 
 #include "base/time/time.h"
+#include "bat/ads/ads.h"
```

Comment:
@tmancey do we need to add now, since `catalog.cc` was relying on it transitively via `ads_impl.h` (which we now removed?)

Reply:
Unfortunately yes as we need access to the catalog filename in ads.h